sakura_core/CSelectLang.cpp

```diff
@@ -362,12 +362,11 @@ int CLoadString::CLoadStrBuffer::LoadString( UINT uid )
 	return nRet;
 }
 
-void CSelectLang::ChangeLang( WCHAR* pszDllName )
+void CSelectLang::ChangeLang( const WCHAR* pszDllName )
 {
 	/* 言語を選択する */
-	UINT unIndex;
-	for ( unIndex = 0; unIndex < CSelectLang::m_psLangInfoList.size(); unIndex++ ) {
-		CSelectLang::SSelLangInfo* psLangInfo = CSelectLang::m_psLangInfoList.at( unIndex );
+	for ( UINT unIndex = 0; unIndex < CSelectLang::m_psLangInfoList.size(); unIndex++ ) {
+		const CSelectLang::SSelLangInfo* psLangInfo = CSelectLang::m_psLangInfoList.at( unIndex );
```

Comment:
(対応不要) 記述が冗長すぎるのをどこかでなんとかしたいような。

```suggestion
		const auto psLangInfo = m_psLangInfoList.at( unIndex );
```

拡張for使えば `std::vector<T>::at(n)` を呼ばなくて済みます。

あとたぶん、このファイル `CSelectLang.cpp` の存在自体が冗長。